dev-java/openjdk/openjdk-17.0.1_p12.ebuild

```diff
@@ -155,12 +188,12 @@ src_configure() {
 		--with-extra-cflags="${CFLAGS}"
 		--with-extra-cxxflags="${CXXFLAGS}"
 		--with-extra-ldflags="${LDFLAGS}"
-		--with-freetype=system
-		--with-giflib=system
-		--with-harfbuzz=system
-		--with-lcms=system
-		--with-libjpeg=system
-		--with-libpng=system
+		--with-freetype="${I_KNOW_WHAT_I_AM_DOING_BOOTSTRAP:-system}"
+		--with-giflib="${I_KNOW_WHAT_I_AM_DOING_BOOTSTRAP:-system}"
+		--with-harfbuzz="${I_KNOW_WHAT_I_AM_DOING_BOOTSTRAP:-system}"
+		--with-lcms="${I_KNOW_WHAT_I_AM_DOING_BOOTSTRAP:-system}"
+		--with-libjpeg="${I_KNOW_WHAT_I_AM_DOING_BOOTSTRAP:-system}"
+		--with-libpng="${I_KNOW_WHAT_I_AM_DOING_BOOTSTRAP:-system}"
```

Comment:
maybe change it so something shorter?
if's ok name as 1 time use, but this looks UUUGLY.

maybe `${XPAK_BOOTSTRAP:-system}` ?